Userland/Libraries/LibWeb/Geometry/DOMMatrixReadOnly.cpp

```diff
@@ -781,4 +789,63 @@ WebIDL::ExceptionOr<void> validate_and_fixup_dom_matrix_init(DOMMatrixInit& init
     return {};
 }
 
+// https://drafts.fxtf.org/geometry/#parse-a-string-into-an-abstract-matrix
+WebIDL::ExceptionOr<Tuple<Gfx::DoubleMatrix4x4, bool>> parse_dom_matrix_init_string(JS::Realm& realm, String const& transform_list)
+{
+    // 1. If transformList is the empty string, set it to the string "matrix(1, 0, 0, 1, 0, 0)".
+    auto const& transform_list_non_empty = transform_list.is_empty() ? "matrix(1, 0, 0, 1, 0, 0)"_string : transform_list;
```

Comment:
```suggestion
    auto const transform_list_non_empty = transform_list.is_empty() ? "matrix(1, 0, 0, 1, 0, 0)"_string : transform_list;
```

_technically_ fine as is, but did make me do a double take as the string is a temporary

Reply:
Does it even need to be a `String`? It's just passed to `parse_css_value`, which takes a `StringView`.
```c++
WebIDL::ExceptionOr<Tuple<Gfx::DoubleMatrix4x4, bool>> parse_dom_matrix_init_string(JS::Realm& realm, StringView transform_list)
{
    // 1. If transformList is the empty string, set it to the string "matrix(1, 0, 0, 1, 0, 0)".
    if (transform_list.is_empty())
        transform_list = "matrix(1, 0, 0, 1, 0, 0)"sv;
```

Reply:
@bplaat You can just pass the `StringView` by value (see https://github.com/SerenityOS/serenity/commit/8b1108e4858f797c9216dc8ae4a3918ad50c73b4 - we do not pass `StringView` by const-ref)  and then you can implement the empty check like above.